git_machete/cmd.py

```diff
@@ -228,7 +221,7 @@ def run_cmd(cmd, *args, **kwargs):
     mark_current_directory_as_possibly_non_existent()
 
     if opt_debug and exit_code != 0:
-        sys.stderr.write(dim("<exit code: %i>\n\n" % exit_code))
+        sys.stderr.write(dim(f"<exit code: {exit_code:d}>\n\n"))
```

Comment:
Actually... what happens if you skip the `:d` for number-typed values? does it make any difference?

Reply:
no diff. redundant, dropped